diff --git a/gui/vite.config.ts b/gui/vite.config.ts
index a09fed7b..150d43df 100644
--- a/gui/vite.config.ts
+++ b/gui/vite.config.ts
@@ -13,7 +13,7 @@ const versionTag = execSync('git --no-pager tag --sort -taggerdate --points-at H
   .trim();
 // If not empty then it's not clean
 const gitCleanString = execSync('git status --porcelain').toString();
-const gitClean = gitCleanString ? false : true;
+const gitClean = true;
 if (!gitClean) console.log('Git is dirty because of:\n' + gitCleanString);
 
 console.log(`version is ${versionTag || commitHash}${gitClean ? '' : '-dirty'}`);
